pootle/apps/pootle_store/models.py

```diff
@@ -214,10 +219,22 @@ class Unit(models.Model, base.TranslationUnit):
     target_length = models.SmallIntegerField(db_index=True, default=0,
                                              editable=False)
 
-    developer_comment = models.TextField(null=True, blank=True)
-    translator_comment = models.TextField(null=True, blank=True)
-    locations = models.TextField(null=True, editable=False)
-    context = models.TextField(null=True, editable=False)
+    developer_comment = models.TextField(
+        null=True,
+        blank=True,
+        validators=[MaxLengthValidator(4096)])
+    translator_comment = models.TextField(
+        null=True,
+        blank=True,
+        validators=[MaxLengthValidator(4096)])
```

Comment:
I would further restrict this. 4096 characters for a translator comment is like a whole novel.